lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -395,34 +408,54 @@ namespace {
     return Call.get();
   }
 
-  void ValueExtractionSynthesizer::FindAndCacheRuntimeDecls() {
+  static bool VSError(clang::Sema* Sema, clang::Expr* E, llvm::StringRef Err) {
```

Comment:
1. I don't want any errors suppressed. Even if setting LookupResult mostly catches the errors, I know BuildDeclarationNameExpr will in some cases suppress them anyway.
2. LookupResult is being passed an invalid source location which makes any diagnostics generated poor.
3. Yes there is a conflict, which will alleviate 2, but 1 will still stand.